libraries/AP_Camera/AP_Camera.h

```diff
@@ -68,6 +69,11 @@ class AP_Camera {
 
     static const struct AP_Param::GroupInfo        var_info[];
 
+    // update flight control mode. The control mode is vehicle type specific
+    //void update_control_mode(uint8_t mode) { _control_mode = mode; }
```

Comment:
Dead line that I think was meant to be removed? (And the comment above needs to be updated)